src/issuance.rs

```diff
@@ -1183,17 +1192,17 @@ mod tests {
 
         let wrong_isk: IssuanceAuthorizingKey = IssuanceAuthorizingKey::random(&mut rng);
 
-        let mut signed = bundle.prepare(sighash).sign(rng, &isk).unwrap();
+        let mut signed = bundle.prepare(sighash).sign(&isk).unwrap();
 
         signed.set_authorization(Signed {
-            signature: wrong_isk.sign(&mut rng, &sighash),
+            signature: wrong_isk.sign(&sighash),
         });
 
         let prev_finalized = &HashSet::new();
 
         assert_eq!(
             verify_issue_bundle(&signed, sighash, prev_finalized).unwrap_err(),
-            IssueBundleInvalidSignature(InvalidSignature)
+            IssueBundleInvalidSignature(schnorr::Error::new())
```

Comment:
Here, I'm using the `schnorr::Error` without any `source` details. Suggestions appreciated.

Reply:
Done